### DISCUSSION TITLE
Can we please add support for OpenEthereum again

### DISCUSSION THREAD

**Discussion by aacostarua** [blockscout/blockscout#9792] [Support]
As I understand v5.3.2, the OpenEthereum config was switched under Nethermind, yet it doesn't work. Is there anyway to keep the parity old code, even if you remove it from the config on your site.  I would still really appreciate being able to use the latest up to date Blockscout versions for our private project running on the last version of OpenEthereum.  Thank you guys!

**Comment by GimluCom** [comment]
>  yet it doesn't work

Could you please provide more details on the issue?
